src/main/java/mods/railcraft/Railcraft.java

```diff
@@ -133,4 +136,9 @@ public void handleMinecartInteract(MinecartInteractEvent event) {
     event.setCanceled(this.minecartHandler.handleInteract(cart, player));
     event.setCanceled(this.crowbarHandler.handleInteract(cart, player, hand));
   }
+
+  @SubscribeEvent
+  public void particleRegistration(ParticleFactoryRegisterEvent event) {
```

Comment:
Move this into `ClientDist` because it's client side only. Also, move the contents of the register method into the event handler.